CondCore/PhysicsToolsPlugins/plugins/DropBoxMetadata_PayloadInspector.cc

```diff
@@ -175,9 +217,15 @@ namespace {
 
       l_theDisplay.printDiffWithMetadata(f_theRecordMap);
 
-      TCanvas canvas("Canv", "Canv", 1200, 100 * std::max(f_records.size(), l_records.size()));
-      std::string fileName(this->m_imageFileName);
```

Comment:
I am afraid you need to percolate this within `DBMetaDataPlotDisplay` and use it to save the result png file otherwise the PayloadInspector instance of the ConDB Browser won't be able to fetch the plot to be displayed.